### PR TITLE
Warn when media is set to autoplay

### PR DESCRIPTION
**Pull request by devindford**
## Types of changes
- [ ] Bug fix (a non-breaking change which fixes an issue)
- [X] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Description
Checka11y.css will check for any `<video autoplay></video>` tags and warn users if they are applied by surrounding the video in the red warning border.
Resolves: #18 

## Link(s)
Link: [a11y project documentation on autoplay](https://www.a11yproject.com/checklist/#make-sure-that-media-does-not-autoplay)

## Screenshot(s)
Autoplay test screenshot

## Checklist:

- [X] I have thoroughly read the [CONTRIBUTING](../CONTRIBUTING.md) guidelines.
- [X] I understand my pull request will be thoroughly reviewed at high detail.
- [X] I understand the work in my pull request will **only** be available in the next version release of Checka11y.css and not in the current version release.
- [X] I confirm the work in this pull request is valid according to my findings and is not something for anything personal.
- [X] I have updated the [README](../README.md) where and if applicable (still put an `x` if you have considered this but thought there was nothing to add or modify).
- [X] I have added myself to the `contributors` section in `package.json` (still put an `x` if you have considered this but decided not to add yourself).
- [X] I have checked I have **not** committed any **accidental** files.
- [ ] I have tested all the main modern browsers (I.e. Chrome, Firefox, Edge, Safari - please leave this unchecked if there were any browsers listed you could not test and list them in the [help](#help) section with details why you couldn't test that browser)
- [X] All new and existing a11y checks still work correctly (compare your local `test/index.html` to the `test/index.html` in the `master` branch).

## Help

Unable to test on safari and edge due to running Linux

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -57,6 +57,7 @@ A list of every a11y concern Checka11y.css will check for and highlight:
   - `<textarea>`
   - `<video controls>`
 - Checks the `dir` attribute is only set to `ltr`, `rtl` or `auto`
+- Checks the `video` attribute to ensure `autoplay` is not set
```
Comment (jackdomleo7) [inline README.md:60]: Nice work! Could we change one thing? Could we remove the `video` check? We'd like to check for whether `autoplay` exists anywhere, such as `video` and `audio`. So I think the below will be ok:

```css
[autoplay] {
  ...
}
```

**Review by jackdomleo7 — CHANGES_REQUESTED** [review]
Very nice Devin! Could you update with the latest master? And there's one minor change, then this PR will be good.

File: README.md (hunk 1 above)
Comment (devindford) [reply README.md:60]: Ok will change it now!

**Review by jackdomleo7 — APPROVED** [review]
Nice, thank you!!

**Pull request merged**